### PR TITLE
add request throttle example to README

### PR DESCRIPTION
**Pull request by Flet**
I thought this might be useful for other folks!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -276,6 +276,36 @@ const opts = {
 get(opts, function (err, res) {})
 ```
 
+### Throttle requests
+
+You can use [limiter](https://github.com/jhurliman/node-rate-limiter) to throttle requests. This is useful when calling an API that is rate limited.
+
+```js
+const simpleGet = require('simple-get');
```
Comment (feross) [inline README.md]: Can you make the example code follow `standard`?

File: README.md [hunk 2]
```diff
@@ -276,6 +276,36 @@ const opts = {
...
+// covers both get() and get.concat()
+const get = (opts, cb) => limiter.removeTokens(1, () => simpleGet(opts, cb));
+get.concat = (opts, cb) => limiter.removeTokens(1, () => simpleGet.concat(opts, cb));
+
+const opts = {
+  url: 'http://example.com'
+};
+
+get(opts, processResult);
+get.concat(opts, processResult);
```
Comment (feross) [inline README.md]: Can you just use one of the methods here? I think that would make the example clearer. It's fine to rate limit both methods, but I would just use one them, say `get.concat`.

File: README.md (hunk 1 above)
Comment (Flet) [inline README.md]: oops, yep! 😊

**Comment by Flet** [comment]
better?

**Comment by feross** [comment]
Looks good! Sorry for being pedantic!

**Comment by Flet** [comment]
No worries, its was great feedback :)

I flip between standard and semistandard in my editor as I switch projects. It happened to be on `semistandard` when I wrote the example heh. I smash the "fix" button as I write stuff and I don't worry unless there are errors.

**Comment by Flet** [comment]
Aww thanks !

**Pull request merged**